### DISCUSSION TITLE
Research and overview: How do paradigms arise in society and get adopted?

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#72] [Research]
This is a key question in our core SCQH issue tree / theory of change.

## Acceptance

* [x] Key references in a zotero collection **Started under 'Social paradigm shifts'**
* [x] Summary of key ideas / terms **Preliminary doc started here: https://docs.google.com/document/d/1nAVFTN8SIEGWU6Mf-fMVN4Sjn2n2DU0tdxsld7O1cQI/edit?usp=sharing**
* [x] Identification of next steps **Turn the current material into a blog post and turn this issue into a discussion with a link to what we have so far**
* [x] Initial essay / post **https://lifeitself.us/2021/10/25/social-paradigm-shifts-a-pre-survey-of-the-literature/**
* [x] A discussion in the forum here (either convert this or start anew with the same question and post up context and links)

## Tasks

* [x] Key references in a zotero collection **Started under 'Social paradigm shifts'**
* [x] Summary of key ideas / terms **Preliminary doc started here: https://docs.google.com/document/d/1nAVFTN8SIEGWU6Mf-fMVN4Sjn2n2DU0tdxsld7O1cQI/edit?usp=sharing**
* [x] Identification of next steps **Turn the current material into a blog post and turn this issue into a discussion with a link to what we have so far**
* [x] (Initial essay / post **https://lifeitself.us/2021/10/25/social-paradigm-shifts-a-pre-survey-of-the-literature/**

**Comment by rufuspollock** [comment]
@theo-cox can you drop a quick summary comment of what you have done so far and what we do next.

**Comment by theo-cox** [comment]
@rufuspollock have begun to capture some of the core fields relevant to this topic, some major themes sitting under these fields and a reading list in Zotero. From this I've also added some preliminary reflections at the top of the document. Next steps would be as follows:
1) tidy the notes a bit
2) use this to construct a research agenda/plan, including research questions and some direction around how we should focus on individual literatures and their overlaps

We may want to return to this, and indeed I think it would be highly valuable, but note that this would be a huge research project - this question would be PhD worth in any one of the fields listed, let alone synthesising all of them...

**Comment by theo-cox** [comment]
@rufuspollock issue now converted to discussion as per final point
